README.md

```diff
@@ -1,264 +1,135 @@
+
++ [ ] A running [`deployer-concourse` instance from `paas-bootstrap`](https://github.com/alphagov/paas-bootstrap)
++ [ ] Make
++ [ ] Ruby >= 2.7
++ [ ] [GDS CLI](https://github.com/alphagov/gds-cli)
++ [ ] Access to `paas-credentials` (private repository) and tools installed
++ [ ] Connection to GDS VPN
++ [ ] Permission to assume the `Admin` role of the relevant AWS account (dev, ci, staging, production)
++ [ ] `AWS_DEFAULT_REGION` environment set the desired region for the environment
++ [ ] [cf CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html) >=7
+
+### Deploy Cloud Foundry
+
+These instructions contain placeholders where the exact command may vary. The below table explains the purpose of those placeholders
+
+| Placeholder   | Purpose                                                                                                                                                                                                           |
+| ------------- | ------------------------------------------|
+| `$ACCOUNT`    | The AWS account being targeted (for example, `dev`, `staging`)|
+| `$ENV` | The name of the environment being targeted. In the case of short lived development environments, this should have a value of `dev`, and the specific environment is set by the `DEPLOY_ENV` environment variable (max 8 chars)|
+
+1. Log in to [Credhub](https://docs.cloudfoundry.org/credhub/) in the environment by running this and following the instructions on screen. This will take you in to a new shell session.
+   ```shell
+   gds aws paas-$ACCOUNT-admin -- make $ENV credhub
+   ```
+
+1. Upload the secrets to Credhub from the Credhub shell session.
+   ```shell
+   make $ENV upload-all-secrets
+   ```
+
+   Note: you do not need to use GDS CLI here because the Credhub shell session contains the AWS credentials in environment variables
+
+1. Exit the Credhub shell session
+
+1. Deploy the pipeline configurations using `make`. This will upload/update the pipelines. Select the target based on which AWS account you want to work with:
+
+   ```shell
+   gds aws paas-$ACCOUNT-admin -- make $ENV pipelines
+   ```
+1. Log in to Concourse. See the [Accessing Concourse](#accessing-concourse).
+
+1. Tun the `generate-paas-admin-git-keys`, `generate-paas-billing-git-keys` and `generate-paas-aiven-broker-git-keys` jobs in the job group `operator`. This will generate and store some SSH keys needed by other jobs.
+
+1. Run the `create-cloudfoundry` pipeline, starting from the left-hand `pipeline-lock` job. This will configure and deploy Cloud Foundry. It might take a couple of hours to complete.
+
+## Cloud Foundry deployment configuration options
+
+There are a handful of configuration options which can change a Cloud Foundry deployment which can only be set at pipeline level. Each of the properties in the below table should be set as Make variables when setting the pipelines
```

Comment:
Should we have the backticks around 'make' here?

Reply:
I'm not sure what the rule are here. Make is the name of the product, and `make` is the name of the program.

Reply:
It should be OK as it is, then. Perhaps we could link to the product's docs here?

Reply:
Good idea!